app/application.js

```diff
@@ -29,8 +29,14 @@ class RunConditionTableApplication {
         this.loggedUsers = {
             tokenToUserData: {},
         };
+
+        const args = process.argv.slice(2);
+        if (args[0] === 'noid') {
```

Comment:
As you already have the "config.js" file imported, it should be as simple as:
 
```suggestion
        if (!config.openId) {
```